dwave/preprocessing/composites/spin_reversal_transform.py

```diff
@@ -25,15 +25,32 @@
 class SpinReversalTransformComposite(dimod.core.Sampler, dimod.core.Composite):
     """Composite for applying spin reversal transform preprocessing.
 
-    Spin reversal transforms (or "gauge transformations") are applied
-    by randomly flipping the spin of variables in the Ising problem. After
-    sampling the transformed Ising problem, the same bits are flipped in the
-    resulting sample [#km]_.
+    A spin-reversal transform does not alter the Ising problem; the 
+    transform simply amounts to reinterpreting spin up as spin down, 
+    and visa-versa, for a particular spin. The technique works as follows: 
```

Comment:
I think "does not alter the Ising problem" is confusing/untrue. The problem is altered. I think you can say that the transform is isomorphic.